src/workers/VerovioWorker.js

```diff
@@ -49,4 +39,36 @@ function handleNeonEvent (evt) {
   postMessage(result);
 }
 
-onmessage = handleNeonEvent;
+var Module = {
+  onRuntimeInitialized: () => {
+
+    toolkit = new verovio.toolkit();
+    toolkit.setOptions({
+      from: 'mei',
+      footer: 'none',
+      header: 'none',
+      pageMarginLeft: 0,
+      pageMarginTop: 0,
+      font: 'Bravura',
+      useFacsimile: true,
+      createDefaultSyl: true,
+      createDefaultSylBBox: true,
+      useRotate: true
+    });
+    console.debug("READY");
+    onmessage = handleNeonEvent;
+    for (const message of backlog) {
+      console.log("Handling message...");
+      handleNeonEvent(message);
+    }
```

Comment:
Should this stay in production or is it just for debugging?

Reply:
Just debugging. Nice catch!